app-backend/database/src/main/scala/com/azavea/rf/database/tables/Uploads.scala

```diff
@@ -47,6 +42,97 @@ class Uploads(_tableTag: Tag) extends Table[Upload](_tableTag, "uploads")
   val visibility: Rep[Visibility] = column[Visibility]("visibility")
```

Comment:
I don't agree -- as it stands, `circe` will grab what it can and throw it into scene metadata and pass any other json in that field into the `metadata` column. I think that's useful -- we can support arbitrary KV pair tags _and_ scene-specific metadata!